### PR TITLE
Fixes for brightness automation action

### PR DESCRIPTION
**Pull request by bramkragten**
image
image

### CODE REVIEW

File: src/components/ha-form/ha-form-integer.ts [hunk 1]
```diff
@@ -19,13 +21,16 @@ import "@polymer/paper-input/paper-input";
...
 export class HaFormInteger extends LitElement implements HaFormElement {
   @property() public schema!: HaFormIntegerSchema;
-  @property() public data!: HaFormIntegerData;
-  @property() public label!: string;
-  @property() public suffix!: string;
+  @property() public data?: HaFormIntegerData;
+  @property() public label?: string;
+  @property() public suffix?: string;
+
+  @property() private _checked = false;
```
Comment (balloob) [inline src/components/ha-form/ha-form-integer.ts]: Can't we derive this from the fact that `this.data !== undefined` ?

File: src/components/ha-form/ha-form-integer.ts [hunk 2]
```diff
@@ -39,20 +44,31 @@ export class HaFormInteger extends LitElement implements HaFormElement {
...
             ${this.label}
-            <ha-paper-slider
-              pin=""
-              .value=${this._value}
-              .min=${this.schema.valueMin}
-              .max=${this.schema.valueMax}
-              @value-changed=${this._valueChanged}
-            ></ha-paper-slider>
+            <div class="flex">
+              ${this.schema.optional
```
Comment (balloob) [inline src/components/ha-form/ha-form-integer.ts]: optional and no default value. If there is a default value, we should just render that as that's the value that will go into the system.

Comment (bramkragten) [reply src/components/ha-form/ha-form-integer.ts]: But it is optional? So you should be allowed to not set a value?

Comment (balloob) [reply src/components/ha-form/ha-form-integer.ts]: But if you don't give a value, we will fill in a value. So we should not give the illusion that you can opt-out of giving a value.

**Review by balloob — APPROVED** [review]

File: src/components/ha-form/ha-form-integer.ts (hunk 2 above)
Comment (bramkragten) [reply src/components/ha-form/ha-form-integer.ts]: Ah ok 👍

**Pull request merged**